src/app/(sidebar)/account/styles.scss

```diff
@@ -11,6 +11,14 @@
     align-self: flex-start;
   }
 
+  .Text--dark {
```

Comment:
thoughts @quietbits

Reply:
Looks fine to me, if this is used in many places where a container color isn't inherited. :+1: